code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -921,3 +921,167 @@
 				on = 0
 				spawn(100)
 					on = 1
+
+/////// MANNED TURRET ////////
+
+/obj/machinery/manned_turret
+	name = "machine gun turret"
+	desc = "While the trigger is held down, this gun will redistribute recoil to allow its user to easily shift targets."
+	icon = 'icons/obj/turrets.dmi'
+	icon_state = "machinegun"
+	can_buckle = TRUE
+	density = TRUE
+	max_integrity = 100
+	obj_integrity = 100
+	buckle_lying = 0
+	layer = 4.2

```

Comment:
Defines please, for example, I'm sure 4 is like, ```OBJ_LAYER``` or something, so you'd do ```layer = OBJ_LAYER + 0.2``` so people are aware it's "0.2 layers above the Object layer"